components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/internal/APIManagerComponent.java

```diff
@@ -932,6 +938,18 @@ private void configureNotificationEventPublisher() {
                 adapterParameters.put(APIConstants.PUBLISHING_MODE, APIConstants.NON_BLOCKING);
                 adapterParameters.put(APIConstants.PUBLISHING_TIME_OUT, "0");
                 adapterConfiguration.setStaticProperties(adapterParameters);
+                if (Boolean.parseBoolean(System.getenv("FEATURE_FLAG_REPLACE_EVENT_HUB"))) {
+                    for (String key : eventHubPublisherConfiguration.getProperties().keySet()) {
+                        adapterParameters.put(key, eventHubPublisherConfiguration.getProperties().get(key));
+                    }
+                    adapterParameters.put("is_enabled",
```

Comment:
shall we define a constant for this `is_enabled` parameter name?

Reply:
This is done by https://github.com/wso2/carbon-apimgt/pull/10969